src/main/webapp/src/app/map/map.component.ts

```diff
@@ -28,12 +28,49 @@ export class MapComponent implements AfterViewInit {
   }
 
   mapInitializer(): void {
+    // Coordinates fetched from the user's location to set the center of the map.
     let centerOfMapCoordinates = new google.maps.LatLng(this.location.lat, this.location.lng);
-
     let mapOptions: google.maps.MapOptions = {
       center: centerOfMapCoordinates,
       zoom: 10
     };
     this.map = new google.maps.Map(this.mapContainer.nativeElement, mapOptions);
+    this.loadMarkers();
+  }
+
+  loadMarkers(): void {
+    // Hard-coded dummy markers.
+    const markers = [
+      {
+        position: new google.maps.LatLng( -33.513059, 151.1234),
+        map: this.map,
+        title: "name: Quay restaurant, address: 123 Sydney Street, Sydney, star rating: 3"
+      },
+      {
+        position: new google.maps.LatLng(-33.47,151.865),
+        map: this.map,
+        title: "name: Food Van cafe, address: 123 Sydney Street, Sydney, star rating: 3"
```

Comment:
I was actually thinking of structuring the info window using something like that: https://github.com/helenatran/step/blob/e4e893fd8f5eb9149688d90b476f3829c05ccb9d/portfolio/src/main/webapp/script.js#L321 (this is from my portfolio). 
It's mostly for styling, so if you don't want to bother structuring the info window already, that's totally fine :) We can deal with that later on.

Reply:
Oh whops I misunderstood it